packages/react/src/components/dropdown-list/dropdown-list.tsx

```diff
@@ -66,16 +66,25 @@ const Textbox = styled.div<TextboxProps>`
     height: ${({ $isMobile }) => ($isMobile ? 'var(--size-2halfx)' : 'var(--size-2x)')};
     justify-content: space-between;
     padding: 0 var(--spacing-1x);
+    text-wrap: none;
     user-select: none;
     width: 100%;
     
     ${({ theme }) => focus({ theme }, true)};
 `;
 
+const TextWrapper = styled.span`
```

Comment:
Malheureusement, comme par définition l'overflow est le contenu qui déborde du padding-box, j'ai eu à ajouter un `span` supplémentaire pour que le texte s'interrompe bien avant le début du padding-right. Si quelqu'un a un truc alternatif dans sa manche, je suis preneur!

Reply:
> Malheureusement, comme par définition l'overflow est le contenu qui déborde du padding-box, j'ai eu à ajouter un `span` supplémentaire pour que le texte s'interrompe bien avant le début du padding-right. Si quelqu'un a un truc alternatif dans sa manche, je suis preneur!

J'utilise la meme strategie, je ne connais pas d'autre way.

Pour le nom, j'eviterais le mot Wrapper, j'aurais tendance a garder ca plus lorsqu'on wrap un autre Component. Je le nommerais tout simplement `Text` ou `Label`.

Reply:
_This is the way_